docs/src/main/sphinx/security/file-system-access-control.md

```diff
@@ -379,52 +386,34 @@ Each function rule is composed of the following fields:
   Defaults to `.*`.
 - `function` (optional): regular expression to match against function names.
   Defaults to `.*`.
-- `privileges` (required): zero or more of `EXECUTE`, `GRANT_EXECUTE`.
-
-To explicitly allow the system builtin functions in queries (and SECURITY 
-DEFINER views), you can use the following rule:
-
-```json
-{
-  "functions": [
-    {
-      "catalog": "system",
-      "schema": "builtin",
-      "privileges": [
-        "EXECUTE",
-        "GRANT_EXECUTE"
-      ]
-    }
-  ]
-}
-```
+- `privileges` (required): zero or more of `EXECUTE`, `GRANT_EXECUTE`, `OWNERSHIP`.
 
-Care should be taken when granting permission to the `system` schema of any 
+Care should be taken when granting permission to the `system` schema of a 
 catalog, as this is the schema Trino uses for table function such as `query`.
 These table functions can be used to access or modify the underlying data of
 the catalog.
 
-The following example allows the `admin` user to execute `query` table
-function from any catalog:
+The following example allows the `admin` user to execute `system.query` table function in
+any catalog, and allows all users to create, drop, and execute functions (including
+`SECURITY DEFINER` views) in the `hive.function` schema:
 
 ```json
 {
   "functions": [
-    {
-      "catalog": "system",
-      "schema": "builtin",
-      "privileges": [
-        "EXECUTE",
-        "GRANT_EXECUTE"
-      ]
-    },
     {
       "user": "admin",
       "schema": "system",
       "function": "query",
       "privileges": [
         "EXECUTE"
       ]
+    },
+    {
+      "catalog": "hive",
```

Comment:
```suggestion
      "catalog": "example",
```

Reply:
there are many examples in this doc using a catalog named hive

Reply:
thats fine then .. lets leave it